frontend/src/citizen-frontend/calendar/AbsenceModal.tsx

```diff
@@ -119,6 +120,13 @@ export default React.memo(function AbsenceModal({
     i18n
   )
 
+  const absencesWarning =
+    range.isValid() &&
+    range
+      .value()
+      .complement(reservationsResponse.reservableRange)
```

Comment:
It should be enough to check that `range.start < reservableRange.start`. If you wan to make it fancier, you could also use `!range.intersection(reservationsResponse).isEqual(range)` to avoid adding the `complement()` method.

Reply:
Hmm I don't think so. We need to know if absence period contains more than one nonreservable day. Neither of those answer that AFAIK.